### ISSUE TITLE
DATA deprecated in DRF 3.2.1

### ISSUE THREAD

**Issue by KevinGrahamFoster** [GetBlimp/django-rest-framework-jwt#143]
JSONWebTokenAPIView references request "DATA" which has just been removed from Django Rest framework. I believe the fix is to modify the file to use "data" as follows on like 56:
```
    def post(self, request):
        serializer = self.get_serializer(data=request.data)
```

**Comment by jpadilla** [comment]
Closed via 6d8614268d60f894dad026b5603c8a2fefc35e2f.

**Comment by KevinGrahamFoster** [comment]
Thanks!! :)
